src/modules/Microsoft.FinOpsHubs/secrets/deploy.bicep

```diff
@@ -0,0 +1,62 @@
+/**
+ * Parameters
+ */
+
+ @description('')
+param storageAccountName string
+
+param secretName string
+
+param keyVaultName string
+
+@description('Optional. Azure location where all resources should be created. See https://aka.ms/azureregions. Default: (resource group location).')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry to track anonymous module usage trends, monitor for bugs, and improve future releases.')
+param enableDefaultTelemetry bool = true
+// The last segment of the telemetryId is used to identify this module
+var telemetryId = '00f120b5-2007-6120-0000-40b000000000'
+var finOpsToolkitVersion = '0.0.1'
+
+/**
+ * Resources
+ */
+
+// Telemetry used anonymously to count the number of times the template has been deployed.
+// No information about you or your cost data is collected.
+resource defaultTelemetry 'Microsoft.Resources/deployments@2022-09-01' = if (enableDefaultTelemetry) {
```

Comment:
nit: I don't think we want to use our custom telemetry since that's used to track the toolkit all-up. We can customize this if we want to track this module separately. Not sure it's really needed, tho. Thoughts?

Reply:
Done.